scripts/base/frameworks/spicy/init-framework.zeek

```diff
@@ -47,12 +47,18 @@ export {
 
 # Marked with &is_used to suppress complaints when there aren't any
 # Spicy file analyzers loaded, and hence this event can't be generated.
-# The attribute is only supported for Zeek 5.0 and higher.
 event spicy_analyzer_for_mime_type(a: Files::Tag, mt: string) &is_used
     {
     Files::register_for_mime_type(a, mt);
     }
 
+# Marked with &is_used to suppress complaints when there aren't any
+# Spicy protocol analyzers loaded, and hence this event can't be generated.
```

Comment:
Mostly nit, if we were to get a handle to the event handler via `zeek::event_registry->Register("spicy_analyzer_for_port", false /*is_from_script*/);` unconditionally during `InitPostScript()` or so, wouldn't need the `&is_used` annotation.

Reply:
Yeah, seems fine to me as it is as well though.